finagle-benchmark/src/main/scala/com/twitter/finagle/loadbalancer/RoundRobin.scala

```diff
@@ -0,0 +1,84 @@
+package com.twitter.finagle.loadbalancer
+
+import com.twitter.finagle.{ClientConnection, NoBrokersAvailableException, Service, ServiceFactory}
+import com.twitter.finagle.service.FailingFactory
+import com.twitter.finagle.stats.StatsReceiver
+import com.twitter.finagle.util.OnReady
+import com.twitter.util.{Activity, Future, Promise, Time}
+
+object RoundRobinBalancer {
```

Comment:
Why use P2C with a constant cost per-node?  There's no benefit, right?  RoundRobin is very simple, it probably adds about as much complexity as P2C with constant cost per-node, and it seems easier to reason about.